src/Management/src/Endpoint/ThreadDump/ThreadDumpEndpoint.cs

```diff
@@ -8,27 +8,28 @@
 
 namespace Steeltoe.Management.Endpoint.ThreadDump;
 
-public class ThreadDumpEndpoint : IThreadDumpEndpoint
+internal sealed class ThreadDumpEndpoint : IThreadDumpEndpoint
```

Comment:
I was going to address this separately but since this came up, Il do it now. Besides the shape of the return value, the options paths are also different. However I think we can merge the middleware and endpoints and use a custom converter to do the right thing in the middleware.

Reply:
Nice, thanks!